build/makelib/golang.mk

```diff
@@ -118,36 +118,16 @@ endif
 go.init: $(GLIDE_INSTALL_STAMP)
 	@:
 
-define go.project
-go.build.packages.$(1):
-	@echo === go build $(1) $(PLATFORM)
-	@$(3) $(GO) build -v -i -o $(GO_OUT_DIR)/$(1)$(GO_OUT_EXT) $(4) $(2)
-
-go.build.packages: go.build.packages.$(1)
-
-go.install.packages.$(1):
-	@echo === go install $(1) $(PLATFORM)
-	@$(3) $(GO) install -v $(4) $(2)
-go.install.packages: go.install.packages.$(1)
-endef
-$(foreach p,$(GO_STATIC_PACKAGES),$(eval $(call go.project,$(lastword $(subst /, ,$(p))),$(p),CGO_ENABLED=0,$(GO_STATIC_FLAGS))))
-
-define go.test.project
```

Comment:
we don't use `go.test.project` anymore?

Reply:
it was an intermediate define to run test project builds in parallel. we now run them sequentially so not needed.